tools/python/progress.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+import argparse
+import git
+import os
+import subprocess
+import sys
+from colour import Color
+
+script_dir = os.path.dirname(os.path.realpath(__file__))
+root_dir = os.path.join(script_dir, "..", "..")
```

Comment:
I'm mostly leaving this relatively unmodified from the version in other decomp projects so it's easier to merge changes shold it be necessary. It could be useful for a planned progress API in development from the splat / decomp.me team.